dspace-server-webapp/src/test/java/org/dspace/app/rest/ProcessRestRepositoryIT.java

```diff
@@ -322,6 +324,447 @@ public void getProcessFilesTypesRandomProcessId() throws Exception {
+        Process newProcess2 = ProcessBuilder.createProcess(context, eperson, "mock-script", parameters)
+                                            .withStartAndEndTime("11/01/1990", "19/01/1990").build();
+        Process newProcess3 = ProcessBuilder.createProcess(context, eperson, "mock-script", parameters)
+                                            .withStartAndEndTime("12/01/1990", "18/01/1990").build();
+
+        String token = getAuthToken(admin.getEmail(), password);
+
+        getClient(token).perform(get("/api/system/processes/search/byProperty")
+                                     .param("userId", eperson.getID().toString())
+                                     .param("sort", "startTime,desc"))
+                        .andExpect(status().isOk())
+                        .andExpect(jsonPath("$._embedded.processes", contains(
+                            ProcessMatcher.matchProcess(newProcess3.getName(),
+                                                        String.valueOf(eperson.getID().toString()),
+                                                        newProcess3.getID(), parameters, ProcessStatus.SCHEDULED),
+                            ProcessMatcher.matchProcess(newProcess2.getName(),
+                                                        String.valueOf(eperson.getID().toString()),
+                                                        newProcess2.getID(), parameters, ProcessStatus.SCHEDULED),
+                            ProcessMatcher.matchProcess(newProcess1.getName(),
+                                                        String.valueOf(eperson.getID().toString()),
+                                                        newProcess1.getID(), parameters, ProcessStatus.SCHEDULED)
+                        )))
+                        .andExpect(jsonPath("$.page", is(
+                            PageMatcher.pageEntryWithTotalPagesAndElements(0, 20, 1, 3))));
+    }
+
+    @Test
+    public void searchProcessTestByUserSortedOnNonExistingIsSortedAsDefault() throws Exception {
+        Process newProcess1 = ProcessBuilder.createProcess(context, eperson, "mock-script", parameters)
+                                            .withStartAndEndTime("10/01/1990", "20/01/1990").build();
+        Process newProcess2 = ProcessBuilder.createProcess(context, eperson, "mock-script", parameters)
+                                            .withStartAndEndTime("11/01/1990", "19/01/1990").build();
+        Process newProcess3 = ProcessBuilder.createProcess(context, eperson, "mock-script", parameters)
+                                            .withStartAndEndTime("12/01/1990", "18/01/1990").build();
+
+        String token = getAuthToken(admin.getEmail(), password);
+
+        getClient(token).perform(get("/api/system/processes/search/byProperty")
+                                     .param("userId", eperson.getID().toString())
+                                     .param("sort", "eaz,desc"))
```

Comment:
isn't an invalid search option supposed to throw an exception?

Reply:
I've indeed altered this to throw an exception instead in the tests: https://github.com/DSpace/DSpace/pull/2931/commits/13b59f9522c752f66455f2b83d6f3f67971c81fb#diff-5c0eb5afe2a696f9d1a29faffc28ccc2R782